File: game_bizModule/src/com/game/bizModule/human/msg/GCCreateHuman.java
package com.game.bizModule.human.msg;

import com.game.bizModule.global.AllMsgSerialUId;
import com.game.gameServer.msg.AbstractGCMsgObj;
import com.game.part.msg.type.MsgBool;

/**
 * 创建角色 GC 消息
 *
 * @author hjj2017
 * @since 2015/7/12
 *
 */
public class GCCreateHuman extends AbstractGCMsgObj {
    /** 建角成功? */
    public MsgBool _success;

    /**
     * 类参数构造器
     *
     * @param success
     *
     */
    public GCCreateHuman(boolean success) {
        this._success = new MsgBool(success);
    }

    @Override
    public short getSerialUId() {
        return AllMsgSerialUId.GC_CREATE_HUMAN;
    }
}

File: client_wpf/Xgame.GameBizModule/Human/Msg/GCCreateHuman.cs
using System;

using Xgame.GameBizModule.Global;
using Xgame.GameClient.Msg;
using Xgame.GamePart.Msg.Type;

namespace Xgame.GameBizModule.Human.Msg
{
    /// <summary>
    /// 创建角色完成消息
    /// </summary>
    public class GCCreateHuman : BaseGCMsg
    {
        /** 建角成功? */
        public MsgBool _success;
        /** 失败消息 */
        public MsgStr _errorMsg;

        // @Override
        public override short MsgSerialUId
        {
            get
            {
                return AllMsgSerialUId.CG_CREATE_HUMAN;
            }
        }
    }
}
